### DISCUSSION TITLE
how to use flap control in rosco controller

### DISCUSSION THREAD

**Discussion by ietqlw** [NatLabRockies/ROSCO#182] [General]
Dear all，

I found trailing edge flap control parameters in the rosco.dll. The input channels of the S-function of FAST  nonlinear model in rosco Simulink demo also contain flap control angles. 

How can I use the flap control？

Shall I make any modification on Fast S-Func or source code，or aerodynamic input files？

**Comment by dzalkind** [comment]
I recommend you check out this example: https://github.com/NREL/ROSCO/blob/main/Examples/example_10.py 

And this test case: https://github.com/NREL/ROSCO/tree/main/Test_Cases/BAR_10 

I'm not as familiar with the Simulink setup, but I imagine it's similar to the ROSCO library.

I'm not sure this is an issue in the code, so I'll convert it to a discussion.
